packages/viz/src/BaseChart/BaseChart.tsx

```diff
@@ -29,7 +29,7 @@ export const HvBaseChart = forwardRef<ReactECharts, HvBaseChartProps>(
   (props, ref) => {
     const { option, width, height } = props;
 
-    const { theme } = useVizTheme();
+    const { theme, activeTheme } = useVizTheme();
```

Comment:
Why do we need this? Isn't `activeTheme` the same value as the one provided by `useTheme`? We could just use it instead
```suggestion
    const { theme } = useVizTheme();
    const { activeTheme } = useTheme();
```